vendor/timekeep.te

```diff
@@ -11,4 +11,4 @@ allow timekeep timekeep_vendor_data_file:file create_file_perms;
 
 set_prop(timekeep, timekeep_prop)
 
-r_dir_file(timekeep, sysfs_msm_subsys)
+r_dir_file(timekeep, sysfs_rtc)
```

Comment:
It still wants to `search` `sysfs_msm_subsys` dirs though

Reply:
Seems not sufficient to satisfy me xD please show me the denial

Reply:
Might have been only the java app then, will re-check

Reply:
`avc: granted { search } for comm="timekeep" name="400f000.qcom,spmi" dev="sysfs" ino=19045 scontext=u:r:timekeep:s0 tcontext=u:object_r:sysfs_msm_subsys:s0 tclass=dir`

Might be because `/sys/class/rtc/rtc0/*` are symlinks into `/sys/devices/platform/soc/...` dirs and it has to traverse parents to do that? Either way, it is necessary for timekeep.c as well.
Add `allow timekeep sysfs_msm_subsys:dir search;`

Reply:
Argh, no leave it as it is. The msm_subsys is because the /sys/class/rtc folder isn't labeled, only its subfolders
https://github.com/sonyxperiadev/device-sony-sepolicy/blob/master/vendor/file_contexts#L116
I will correct this

Reply:
Please leave this line as it is so we don't get merge conflicts, I've fixed it properly in https://github.com/sonyxperiadev/device-sony-sepolicy/commit/110cfb465baeb26c52cf0684cd66217fbfd3f9fe#diff-d5b56ef65020cc141928eee913cc89c2R21